### PR TITLE
Add stream endpoint

### PR DESCRIPTION
**Pull request by annaguri**

### CODE REVIEW

File: integrations/extensions/starter-kits/testitall/mock-server/server/controllers/test-controller.js [hunk 1]
```diff
@@ -84,12 +174,6 @@ export function contextAlmostTooLargeTest(req, res) {
   return res.status(200).send(response);
 }
 
-export function successfulPostWithNonJSONResponse(req, res) {
```
Comment (jfmartinez) [inline integrations/extensions/starter-kits/testitall/mock-server/server/controllers/test-controller.js:87]: Not sure why this got deleted in your PR?

Comment (annaguri) [reply integrations/extensions/starter-kits/testitall/mock-server/server/controllers/test-controller.js:87]: Interesting. I rebased before pushing the changes. Is it possible that it got deleted during the rebase? When I ran `difftool`, I only saw the new files/functions. Anyway, I'll compare it to the latest version in `master` and fix it.

File: integrations/extensions/starter-kits/testitall/mock-server/server/controllers/test-controller.js [hunk 2]
```diff
@@ -32,6 +34,94 @@ export function postTest(req, res) {
...
+      }
+      res.write(`data: ${JSON.stringify(chunk)}\n\n`);
+      pieceIndex++;
+    } else {
+      pieceIndex = 0;    
+    }
+  }, 100);
+
+  // Stop streaming after 40 seconds
+  const timeoutId = setTimeout(() => {
```
[the remark this answers is not in the document]
Comment (annaguri) [reply integrations/extensions/starter-kits/testitall/mock-server/server/controllers/test-controller.js:113]: The server will stream for 51 seconds and will mark the response as closed either when the timer expires or when the connection closes, whichever occurs first.

**Review by jnxyp — CHANGES_REQUESTED** [review]
Tried to upload the updated actions.json, but although tooling says upload is successful, **the action list is not actually updated**. I tried the action.json on master branch, and it works okay. I'm looking at the difference between old and new json, but found nothing yet.

Here is the action.json zip I tried to upload:
[action-skill.zip](https://github.com/user-attachments/files/16802995/action-skill.zip)


Also, could you please add the new actions to the list in readme?
https://github.com/annaguri/assistant-toolkit/tree/add_stream_endpoint/integrations/extensions/starter-kits/testitall#wa-actions-and-corresponding-openapi-endpoints

File: integrations/extensions/starter-kits/testitall/testitall.actions.json [hunk 3]
```diff
@@ -150,9 +145,9 @@
                 "type": "integration_interaction",
                 "method": "POST",
                 "internal": {
-                  "spec_hash_id": "837b4f9e9cbcd25948117367cbee4ddcbf870a48496df7e15ad127852843499f",
-                  "match_scenario": 12,
-                  "catalog_item_id": ""
+                  "spec_hash_id": null,
```
Comment (jfmartinez) [inline integrations/extensions/starter-kits/testitall/testitall.actions.json:148]: The `spec_hash_ids` cause the existing extensions to fail, can you recreate this Action Skills JSON with the hash IDs?

Comment (annaguri) [reply integrations/extensions/starter-kits/testitall/testitall.actions.json]: done

**Pull request merged**